InnerEye/train_variables.yml

```diff
@@ -9,8 +9,8 @@ variables:
   datasets_container: 'datasets'
   resource_group: 'InnerEye-DeepLearning'
   docker_shm_size: '440g'
-  node_count: 1
-  workers_per_node: 1
+  node_count: 2
```

Comment:
Personally I think node_count is clearer (because if you are running this on a single VM not in a cluster then cluster_nodes might be confusing), but I will change if you feel strongly about this

Reply:
we can close